### PR TITLE
New serverless pattern- bedrock-lambda-cdk-dotnet

### PR DESCRIPTION
**Pull request by PankajRawat333**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: bedrock-lambda-cdk-dotnet/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Extract text from images using Amazon Bedrock
```
Comment (bfreiberg) [inline bedrock-lambda-cdk-dotnet/README.md:1]:
```suggestion
# Automatically extract text from uploaded images using Amazon Bedrock
```

File: bedrock-lambda-cdk-dotnet/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+This pattern shows how to extract text from images using Amazon Bedrock.
```
Comment (bfreiberg) [inline bedrock-lambda-cdk-dotnet/README.md:3]:
```suggestion
This pattern shows how to automatically extract text from images uploaded to Amazon S3 using Amazon Bedrock.
```

File: bedrock-lambda-cdk-dotnet/example-pattern.json [hunk 3]
```diff
@@ -0,0 +1,59 @@
+{
+    "title": "Extract text from images using Amazon Bedrock",
```
Comment (bfreiberg) [inline bedrock-lambda-cdk-dotnet/example-pattern.json:2]:
```suggestion
    "title": "Automatically extract text from images in Amazon S3 using Amazon Bedrock",
```

File: bedrock-lambda-cdk-dotnet/example-pattern.json [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    "description": "This CDK application demonstrates how to use Amazon Bedrock service to extract text from images.",
```
Comment (bfreiberg) [inline bedrock-lambda-cdk-dotnet/example-pattern.json:3]:
```suggestion
    "description": "This pattern demonstrates how to use Amazon Bedrock to automatically extract text from images uploaded to Amazon S3.",
```

File: bedrock-lambda-cdk-dotnet/example-pattern.json [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    "language": ".NET",
+    "level": "200",
+    "framework": "CDK",
+    "introBox": {
+      "headline": "How it works",
+      "text": [
+        "This pattern shows how to extract text from images using Amazon Bedrock.",
+        "When images are uploaded into the Amazon S3 bucket, notifications are sent to a Lambda function. The Lambda function utilizes that notification to identify the newly uploaded image and sends it to the Amazon Bedrock service, which extracts text from the image and stores it in the S3 bucket."
```
Comment (bfreiberg) [inline bedrock-lambda-cdk-dotnet/example-pattern.json:11]:
```suggestion
        "When images are uploaded into the Amazon S3 bucket, notifications are sent to a Lambda function. The Lambda function utilizes that notification to identify the newly uploaded image and sends it to the Amazon Bedrock, which extracts text from the image and stores it in the S3 bucket."
```

File: bedrock-lambda-cdk-dotnet/README.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+    ```
+    dotnet build
+    ```
+4. Deploy the stack to your default AWS account and region.
+    ```
+    cdk deploy
+    ```
+
+## How it works
+When images are uploaded into the Amazon S3 bucket, notifications are sent to a Lambda function. The Lambda function utilizes that notification to identify the newly uploaded image and sends it to the Amazon Bedrock service, which extracts text from the image and stores it in the S3 bucket.
```
Comment (bfreiberg) [inline bedrock-lambda-cdk-dotnet/README.md:37]:
```suggestion
When images are uploaded into the Amazon S3 bucket, notifications are sent to a Lambda function. The Lambda function utilizes that notification to identify the newly uploaded image and sends it to the Amazon Bedrock, which extracts text from the image and stores it in the S3 bucket.
```

File: bedrock-lambda-cdk-dotnet/README.md (hunk 1 above)
Comment (PankajRawat333) [reply bedrock-lambda-cdk-dotnet/README.md:1]: Hi @bfreiberg you suggested different title for Readme.md and example-pattern.json file, any specific reason?

how about this one
Automatically extract text from uploaded images in Amazon S3 using Amazon Bedrock

File: bedrock-lambda-cdk-dotnet/example-pattern.json (hunk 3 above)
Comment (PankajRawat333) [reply bedrock-lambda-cdk-dotnet/example-pattern.json:2]: I'm only extracting newly uploaded image in S3 bucket. 
this one would be better?
"Automatically extract text from uploaded images in Amazon S3 using Amazon Bedrock."

File: bedrock-lambda-cdk-dotnet/README.md (hunk 1 above)
Comment (bfreiberg) [reply bedrock-lambda-cdk-dotnet/README.md:1]: There is a 75 characters limit for the title in the json file, whereas there is none for the README.

Comment (bfreiberg) [reply bedrock-lambda-cdk-dotnet/README.md:1]: I'm happy with your suggestion for this file "Automatically extract text from uploaded images in Amazon S3 using Amazon Bedrock"

Comment (PankajRawat333) [reply bedrock-lambda-cdk-dotnet/README.md]: fixed

**Comment by PankajRawat333** [comment]
@bfreiberg all code review comment fixed, Could you please approve this

**Pull request merged**